docs/trellis/master/vagrant.md

```diff
@@ -26,13 +26,12 @@ Installation:
 See the [Parallels site](https://parallels.github.io/vagrant-parallels/docs/) for
 more details.
 
-As of January 2022, the Vagrant box needs to be changed as well since the default one doesn't support Ubuntu for ARM yet.
+As of September 2022, the Vagrant box needs to be changed as well since the default one doesn't support Ubuntu for ARM yet.
 
-Update the following two settings in `vagrant.default.yml` (or
+Update the following setting in `vagrant.default.yml` (or
    `vagrant.local.yml`)
 ```yaml
-vagrant_box: 'jeffnoxon/ubuntu-20.04-arm64'
-vagrant_box_version: '>= 1.0.0'
```

Comment:
This still needs to be overwritten due to the default: https://github.com/roots/trellis/blob/7e8e241ce77f033d26d0c6c3aefb491d9dde98f5/vagrant.default.yml#L6

Reply:
Oh oops I didn't notice it was bento now 🤔 I'll have to try this

Reply:
Works ✨